src/babel.js

```diff
@@ -17,23 +18,98 @@ const STYLE_COMPONENT_CSS = 'css'
 export default function ({types: t}) {
   const findStyles = children => (
     children.filter(el => (
```

Comment:
Just a small thing, think you can destructure this param `({node}) => (` which allows you to remove `el` in front of `node`